src/ansys/aedt/core/filtersolutions_core/distributed_geometry.py

```diff
@@ -0,0 +1,263 @@
+        self._dll.setDistributedCapacitorSections.argtype = c_char_p
+        self._dll.setDistributedCapacitorSections.restype = int
+        self._dll.getDistributedCapacitorSections.argtypes = [c_char_p, c_int]
+        self._dll.getDistributedCapacitorSections.restype = int
+
+        self._dll.setDistributedInductorSections.argtype = c_char_p
+        self._dll.setDistributedInductorSections.restype = c_int
+        self._dll.getDistributedInductorSections.argtypes = [c_char_p, c_int]
+        self._dll.getDistributedInductorSections.restype = c_int
+
+        self._dll.setDistributedSplitHeightRatio.argtype = c_char_p
+        self._dll.setDistributedSplitHeightRatio.restype = c_int
+        self._dll.getDistributedSplitHeightRatio.argtypes = [c_char_p, c_int]
+        self._dll.getDistributedSplitHeightRatio.restype = c_int
+
+        self._dll.setDistributedAlternateStubOrientation.argtype = c_bool
+        self._dll.setDistributedAlternateStubOrientation.restype = c_int
+        self._dll.getDistributedAlternateStubOrientation.argtype = POINTER(c_bool)
+        self._dll.getDistributedAlternateStubOrientation.restype = c_int
+
+        self._dll.setDistributedGeometryMaxWidth.argtype = c_char_p
+        self._dll.setDistributedGeometryMaxWidth.restype = c_int
+        self._dll.getDistributedGeometryMaxWidth.argtypes = [c_char_p, c_int]
+        self._dll.getDistributedGeometryMaxWidth.restype = c_int
+
+        self._dll.setDistributedGeometryMinWidth.argtype = c_char_p
+        self._dll.setDistributedGeometryMinWidth.restype = c_int
+        self._dll.getDistributedGeometryMinWidth.argtypes = [c_char_p, c_int]
+        self._dll.getDistributedGeometryMinWidth.restype = c_int
+
+        self._dll.setDistributedGeometryMaxGap.argtype = c_char_p
+        self._dll.setDistributedGeometryMaxGap.restype = c_int
+        self._dll.getDistributedGeometryMaxGap.argtypes = [c_char_p, c_int]
+        self._dll.getDistributedGeometryMaxGap.restype = c_int
+
+        self._dll.setDistributedGeometryMinGap.argtype = c_char_p
+        self._dll.setDistributedGeometryMinGap.restype = c_int
+        self._dll.getDistributedGeometryMinGap.argtypes = [c_char_p, c_int]
+        self._dll.getDistributedGeometryMinGap.restype = c_int
+
+        self._dll.setDistributedApplyLimits.argtype = c_bool
+        self._dll.setDistributedApplyLimits.restype = c_int
+        self._dll.getDistributedApplyLimits.argtype = POINTER(c_bool)
+        self._dll.getDistributedApplyLimits.restype = c_int
+
+        self._dll.setDistributedAdjustLengthOnLimit.argtype = c_bool
+        self._dll.setDistributedAdjustLengthOnLimit.restype = c_int
+        self._dll.getDistributedAdjustLengthOnLimit.argtype = POINTER(c_bool)
+        self._dll.getDistributedAdjustLengthOnLimit.restype = c_int
+
+    @property
+    def fixed_width_capacitor_sections(self) -> str:
+        """Susbtrate width to height ratio of segments and stubs of the translated lumped capacitor sections.
```

Comment:
- Substrate spelling
- Let's reword this comment to make it clearer what the 'fixed width' is (assuming the property name is a good one). The subject of this sentence as it is worded now is the ratio and it's not clear now that relates to 'fixed width capacitor sections'.